OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -62,48 +67,95 @@ public Turreted(ActorInitializer init, TurretedInfo info)
 		{
 			this.info = info;
 			TurretFacing = GetInitialTurretFacing(init, info.InitialFacing);
+
+			if (info.RandomRealign)
+				randomFacing = init.Self.World.SharedRandom.Next(0, 256);
```

Comment:
Probably https://github.com/OpenRA/OpenRA/pull/6503#issuecomment-56280668 still applies then. Alright then... although I'm not certain what is the gain of this realigning logic if it ends up such blatantly stupid.